src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/PanelSummary/index.tsx

```diff
@@ -92,6 +91,28 @@ function PanelSummary({
     }
   };
 
+  /**
+   * Hacky solution that attaches a keyboard shortcut to "shift+c" for adding comments.
+   * TODO: move to a first-class keyboard shortcut hook that we can use to
+   * display a keyboard shortcut modal
+   *
+   */
+  const addCommentRef = useRef<any>(null);
```

Comment:
The ref here is to avoid having a huge dependency chain in `useEffect` below since `addComment` is not memoized. Happy to memoize and add the dependencies for `addComment` if we think this is an issue though!